bin/compile

```diff
@@ -1,22 +1,45 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-indent() {
-  sed -u 's/^/       /'
-}
+# fail fast
+set -e
+
+# debug
+# set -x
+
+# parse and derive params
 
-echo "-----> Installing tippecanoe"
 BUILD_DIR=$1
+CACHE_DIR=$2
+
+function indent() {
+  c='s/^/       /'
+  case $(uname) in
+    Darwin) sed -l "$c";;
+    *)      sed -u "$c";;
+  esac
+}
+
+function error() {
```

Comment:
unused

Reply:
unused in the file i ripped it from too, lol